index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare function abort(): void
+
+export = abort
```

Comment:
I chose `export = abort` instead of `export default` because this warning from [typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html):

> Note that using export default in your .d.ts files requires [esModuleInterop: true](https://www.typescriptlang.org/tsconfig#esModuleInterop) to work. (...). This older syntax is harder to use but works everywhere. Here’s how the above example would have to be written using export=

@kasperisager please confirm if the `export default` syntax is preferred.

Reply:
We should use `export default`, yep, so the declarations match the synthesized ES module.

Reply:
Updated, but yet, as a double check, with this syntax the tool [arethetypeswrong](https://arethetypeswrong.github.io) accuses the error: [Incorrect default export](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseExportDefault.md).

Importing the module locally like we usually do at test files is enough to reproduce the case of `.default` being required:

![image](https://github.com/user-attachments/assets/a4cbaed8-e8df-42c1-baa6-1f8df29834e6)

Reply:
Oh gee... Emits are correct for `export default`, but you're right that it indeed messes with LSP code completion. `export =` it is!